calour/io.py

```diff
@@ -234,7 +236,7 @@ def _read_metadata(ids, f, kwargs):
 def read(data_file, sample_metadata_file=None, feature_metadata_file=None,
          description='', sparse=True, data_file_type='biom',
          sample_metadata_kwargs=None, feature_metadata_kwargs=None,
-         cls=Experiment, *, normalize):
+         cls=Experiment, cut_sample_id_sep=None, rows_are_samples=False, *, normalize):
```

Comment:
I don't like `cut_sample_id_sep` and I don't think we need it. It's only used in the MS read func, right? You can update IDs after read() in read_ms()

Reply:
i don't think we can do it not in the read() function, since the read function links the data and sample_metadata table indices (_read_metadata() line 231, called from read() line 318), and if the indices are not the same (i.e. sample ids haven't been cut, it will not know how to link, and will fail.